build.gradle

```diff
@@ -32,16 +38,7 @@ dependencies {
 
 sourceCompatibility = '1.8'
 group = 'ca.nrc.cadc'
-version = '1011'
-
-ext {
-    intTest_user_name = 'CADCtest'
-    intTest_user_password = 'sywymUL4'
-
-    // Change this value to test other locations, such as Production or RC.
-    intTest_default_web_app_url = 'https://rc-www.canfar.net'
-    intTest_default_web_app_endpoint = '/science-portal'
-}
+version = '1012'
```

Comment:
We don't version the WAR files anymore.

Reply:
ok. removed this and changed the archive name.